std/http/template.ts

```diff
@@ -0,0 +1,111 @@
+import { EntryInfo } from "./file_server.ts";
```

Comment:
@zhmushan Please merge this file into file_server.ts - I don't think the extra module is needed.

Reply:
@ry done